src/orderbook.ts

```diff
@@ -0,0 +1,79 @@
+export class Price {
+  numerator: number;
```

Comment:
Shouldn't this bi a `BN`?

Reply:
By now this has become a fraction class where numerator/denominator are both BN

Reply:
Yes, I saw it later! I like more the new approach!